data.go

```diff
@@ -10,12 +10,13 @@ import (
 
 // TrianglesData specifies a list of Triangles vertices with three common properties:
 // TrianglesPosition, TrianglesColor and TrianglesPicture.
-type TrianglesData []struct {
-	Position  Vec
-	Color     RGBA
-	Picture   Vec
-	Intensity float64
+type vertex struct {
+	Position   Vec
+	Color	   RGBA
+	Picture	   Vec
+	Intensity  float64
 }
+type TrianglesData []vertex
```

Comment:
The reason for this was just that there were two copies of the struct definition, and I hate having two copies of struct definitions. No reason to export it, since it's not visible to anyone else, and no one else needs the type.